src/libraries/System.Runtime.Numerics/tests/BigInteger/GetBitLengthTests.cs

```diff
@@ -39,8 +39,8 @@ public static void RunGetBitLengthTests()
         }
 
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.Is64BitProcess))] // OOM on 32 bit
-        [SkipOnPlatform(TestPlatforms.Browser, "OOM on browser due to large array allocations")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/37093", TestPlatforms.Android)]
+        [OuterLoop("Allocates large arrays")]
+        [SkipOnPlatform(TestPlatforms.AnyUnix, "OOM on browser and mobile due to large array allocations")]
```

Comment:
The message suggests the issue here is constrained environments like browser and mobile, but AnyUnix is going to skip this test everywhere other than Windows.  Is that intentional?  We don't want to run this test on e.g. Linux?

Reply:
I don't think there is much benefit to running it on Linux, Windows, and MacOS. We don't have any OS specific logic here and its really a regression test for an extreme edge case.

Our MacOS pools are limited, so running unnecessary long-running tests there isn't beneficial. Likewise, many external devs use Linux on low-power (raspberry pi) or older devices where resources may be more constrained, so it has a higher risk of negative impact there.

Reply:
Happy to change this to just `iOS | tvOS | Android | Browser` (since we don't have any meta tag) if you think its better to run everywhere though.

Reply:
I think it's better for tests to only be excluded on platforms for which they're not supported / relevant.